### ISSUE TITLE
Flake8 rule 'E999' reported an invalid point

### ISSUE THREAD

**Issue by jarshwah** [AtomLinter/linter-flake8#310]
Flake8 reported an invalid point for the rule `E999`, with the messge `SyntaxError: unexpected EOF while parsing`.

The error is triggered on line 1 of the file. I can't show it all, but it's just basic imports, until the tests begin. Running flake8 on the file from the CLI does not show this error.

```
from django.test import TestCase
from payments.factories import SupplierCreditFactory, PaymentFactory
from orders.factories import InvoiceFactory, InvoiceLineFactory
from core.factories import CompanyFactory
```

Debug information:
Atom version: 1.12.5
Flake8 version: `3.0.4 (pyflakes: 1.2.3, mccabe: 0.5.2, pycodestyle: 2.0.0) CPython 3.5.2 on Darwin`

**Comment by jarshwah** [comment]
Upgrade of flake8 sorted this (as seen in other similar issues).
